pages/attacks/Log_Injection.md

```diff
@@ -54,32 +54,32 @@ from a request object. If the value fails to parse as an integer, then
 the input is logged with an error message indicating what happened.
 
 ```
-    ...
-    String val = request.getParameter("val");
-    try {
-        int value = Integer.parseInt(val);
-    }
-    catch (NumberFormatException) {
-        log.info("Failed to parse val = " + val);
-    }
-    ...
+...
```

Comment:
add java rendering hint?

Reply:
:+1:

Is there any preference for bundling all changes for a file together vs. separating different types of changes?

Reply:
Added :+1:

Reply:
Added :+1: